config.go

```diff
@@ -76,6 +77,72 @@ type DatabaseConfig struct {
 	FederationDB *tapdb.UniverseFederationDB
 }
 
+// UniversePublicAccessStatus is a type that indicates the status of public
+// access to the universe server.
+type UniversePublicAccessStatus string
```

Comment:
Since this is all config related, should this go into the `tapcfg` package? Non-blocking, just kind of weird here as the file is otherwise empty of any logic.

Reply:
I need to use that type in `taproot-assets/config.go`. I'll get a circular dep if it's in `tapcfg`.